docs/sources/whatsnew/whats-new-in-v7-4.md

```diff
@@ -213,6 +213,12 @@ You can now select a font, other than the default, for Unicode-based scripts. As
 
 [Reporting]({{< relref "../enterprise/reporting.md#rendering-configuration" >}}) was updated as a result of this change.
 
+### Request security
+
+Request security introduces ways to limit requests from the Grafana server, and it targets requests that are generated by users.
```

Comment:
```suggestion
Request security introduces new ways to limit requests from the Grafana server, and it targets requests that are generated by users.
```

Reply:
-1

Reply:
Fair enough. Can we improve the sentence then? "introduces ways..." does not read well. Thanks.

Reply:
"Request security limits requests from the Grafana server, and it targets requests that are generated by users."